pyproject.toml

```diff
@@ -50,9 +50,11 @@ coverage = "^7.2.7"
 
 [tool.poetry.group.extras.dependencies]
 google-cloud-aiplatform = "^1.26.1"
+sqlalchemy-databricks = "^0.2.0"
+snowflake-sqlalchemy = "^1.5.0"
```

Comment:
@ArslanSaleem Why are these in the extras and how do the final user installs these?

Reply:
Also, beware that `sqlalchemy-databricks` has been deprecated a while ago (see [here](https://github.com/crflynn/sqlalchemy-databricks)).

Reply:
I think that's the reason of the downgrade, but if an alternative is available we should definitely switch to that.

@ArslanSaleem here's the maintained version, I'll open a issue for that:
https://github.com/databricks/databricks-sql-python